buildpipeline/dotnet-standard-win.json

```diff
@@ -32,28 +32,7 @@
       },
       "inputs": {
         "filename": "build.cmd",
-        "arguments": "-OfficialBuildId=$(OfficialBuildId) -- /p:SignType=$(SignType) /p:MicroBuild_SigningEnabled=true /p:OverridePackageSource=$(PB_RestoreSource)",
```

Comment:
Maybe the diff is bad but it looks like you removed the build command. Have you tested this privately?

Reply:
There used to be one `build.cmd` invocation to build the projects, then another to sign them. Now we can just call `build.cmd -signPackages` to do both. I'll get a private pipebuild going to confirm everything looks good.